main.js

```diff
@@ -1 +1,115 @@
-console.log("Tic Tac Toe")
\ No newline at end of file
+// alternatives for var?
+var grid, playerActive, gameGoing, counts;
+
+function makeArray(w, h, val) {
+  result = [];
+  for (let i = 0; i <= h; i++) {
+    let temp = [];
+    result.push(temp);
+    for (let j = 0; j <= w; j++) {
+      temp.push(val);
+    }
+  }
+  return result;
+}
+
+//Restart everything
+const init = function() {
+  counts = 0;
+  grid = makeArray(2, 2, -1);
+  playerActive = 0;
+  gameGoing = true;
+  document.querySelector(".p" + 0 + "bar").style.display = "block";
+  document.querySelector(".p" + 1 + "bar").style.display = "none";
+  document.querySelector(".w" + 1).style.display = "none";
+  document.querySelector(".w" + 0).style.display = "none";
+
+  let imgs = document.querySelector(".images");
+  while (imgs) {
+    imgs.parentNode.removeChild(imgs);
+    imgs = document.querySelector(".images");
+  }
+  let draw = document.querySelector(".draw");
+  if (draw) {
+    draw.parentNode.removeChild(draw);
+  }
+};
+
+//Updating game score and check for winning condition
+const algorithm = function(j, i) {
+  grid[j][i] = playerActive;
+  //column-wise
+  if (
+    grid.slice(j - 1)[0].slice(i)[0] === playerActive &&
```

Comment:
Overly engineered, making people hard to understand even if the logic itself is not hard